check_staleness.go

```diff
@@ -28,8 +30,37 @@ func getVcsCommand(vendor string, path string) (*vcsCmd, string, error) {
 	return nil, "", fmt.Errorf("Unable to get the VCS")
 }
 
+func locateGomfile() (string, error) {
+	dir, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+
+	for {
+		file := filepath.Join(dir, "Gomfile")
+		if isFile(file) {
+			fmt.Println(file)
+			return file, nil
+		}
+		next := filepath.Clean(filepath.Join(dir, ".."))
+		if next == dir {
+			break
+		}
+		dir = next
+	}
+
+	return "", errors.New("Can't locate Gomfile")
+}
+
 func checkStaleness() error {
-	allGoms, err := parseGomfile("Gomfile")
+	var allGoms []Gom
```

Comment:
Maybe define this just before you use it? It feels cleaner to me, though I'm not sure what is *idiomatic*